features/autofill.yml.dist

```diff
@@ -2,8 +2,9 @@
 # Do not edit this file by hand. Edit the source file instead!
 
 status:
-  baseline: low
+  baseline: high
   baseline_low_date: 2023-02-09
+  baseline_high_date: 2025-08-09
```

Comment:
This one is risky. See https://github.com/mdn/browser-compat-data/issues/26245. I'll hold this back.